app/models/search/collection_search_form.rb

```diff
@@ -0,0 +1,114 @@
+class CollectionSearchForm
+
+  extend ActiveModel::Naming
+  include ActiveModel::Conversion
+  include ActiveModel::Validations
+
+  ATTRIBUTES = [
+    :title,
+    :challenge_type,
+    :signup_open,
+    :unrevealed,
+    :anonymous,
+    :closed,
+    :moderated,
+    :fandom_ids,
+    :owner_ids,
+    :parent_id,
+    :moderator_ids,
+    :signups_open_at,
+    :signups_close_at,
+    :assignments_due_at,
+    :works_reveal_at,
+    :authors_reveal_at,
+    :sort_column,
+    :sort_direction,
+    :page
+  ]
+
+  attr_accessor :options
+
+  ATTRIBUTES.each do |filterable|
+    define_method(filterable) { options[filterable] }
+  end
+
+  def initialize(opts={})
+    @options = opts
+    process_options
+    @searcher = CollectionQuery.new(@options)
+  end
+
+  def process_options
+    @options.delete_if { |k, v| v == "0" || v.blank? }
+    set_sorting
+  end
+
+  def set_sorting
+    @options[:sort_column] ||= default_sort_column
+    @options[:sort_direction] ||= default_sort_direction
+  end
+
+  def persisted?
+    false
+  end
+
+  def summary
+    summary = []
+    if @options[:title].present?
+      summary << "Title: #{@options[:title]}"
+    end
+
+    if @options[:sort_column].present?
+      summary << "sort by: #{name_for_sort_column(@options[:sort_column]).downcase}" +
+        (@options[:sort_direction].present? ?
+          (@options[:sort_direction] == "asc" ? " ascending" : " descending") : "")
+    end
+    summary.join(" ")
+  end
+
+  def search_results
+    @searcher.search_results
+  end
+
+  ###############
+  # SORTING
+  ###############
+
+  SORT_OPTIONS = [
+    ['Date Posted', 'created_at'],
```

Comment:
This should probably be "Date Created" -- "posted" is more specific to works. (+1 to Hound's comment about the quotes.)